src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -135,7 +135,11 @@ public static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, obj
 		void OnQueryTextSubmit(object? sender, QueryTextSubmitEventArgs e)
 		{
 			VirtualView.SearchButtonPressed();
-			e.Handled = true;
+			(sender as SearchView)?.ClearFocus();
```

Comment:
why are we clearing the focus here? 

We typically never want to change the focus

Reply:
Hi @PureWeen  ,In the suggestion provided in the comment
[discussion_r1820700241](https://github.com/dotnet/maui/pull/24198#discussion_r1820700241)
, it was mentioned that we must clear the focus.